contracts/minter-suite/Minters/MinterDAExpRefund/MinterDAExpRefundV0.sol

```diff
@@ -0,0 +1,786 @@
+     * @param _projectId Project ID to set auction details for.
+     */
+    function resetAuctionDetails(uint256 _projectId)
+        external
+        onlyCoreAdminACL(this.resetAuctionDetails.selector)
+    {
+        // CHECKS
+        ProjectConfig storage _projectConfig = projectConfig[_projectId];
+        // once one or more refundable purchase have been made on this minter,
+        // a project's auction cannot be reset due to refunds possible on this minter
+        require(
+            _projectConfig.numRefundableInvocations == 0,
+            "No modifications after refundable purchases"
+        );
+        // EFFECTS
+        // reset to initial values
+        _projectConfig.timestampStart = 0;
+        _projectConfig.priceDecayHalfLifeSeconds = 0;
+        _projectConfig.startPrice = 0;
+        _projectConfig.basePrice = 0;
+
+        emit ResetAuctionDetails(_projectId);
+    }
+
+    /**
+     * @notice This represents an admin stepping in and reducing the sellout
+     * price of an auction. This is only callable by the core admin, only
+     * before the auction is marked as valid (which enables the artist or admin
+     * to initiate admin and artist withdrawals).
+     * This is only intended to be used in the case where for some reason,
+     * whether malicious or accidental, the sellout price was too high.
+     * Examples of this include:
+     *  - The artist reducing a project's maxInvocations on the core contract
+     *    after an auction has started, but before it ends, eliminating the
+     *    ability of purchasers to fairly determine market price under the
+     *    original, expected auction parameters.
+     *  - Any other reason the admin deems to be a valid reason to reduce the
+     *    sellout price of an auction, prior to marking it as valid.
+     * @param _projectId Project ID to reduce auction sellout price for.
+     * @param _newSelloutPrice New sellout price to set for the auction. Must
+     * be less than the current sellout price.
+     */
+    function adminEmergencyReduceSelloutPrice(
+        uint256 _projectId,
+        uint256 _newSelloutPrice
+    )
+        external
+        onlyCoreAdminACL(this.adminEmergencyReduceSelloutPrice.selector)
+    {
+        ProjectConfig storage _projectConfig = projectConfig[_projectId];
+        // @dev no need to check that auction max invocations has been reached,
+        // because if it was, the sellout price will be zero, and the following
+        // check will fail.
+        require(
+            _newSelloutPrice < _projectConfig.selloutPrice,
+            "May only reduce sellout price"
+        );
+        require(
+            _newSelloutPrice >= _projectConfig.basePrice,
+            "May only reduce sellout price to base price or greater"
+        );
+        require(
+            _projectConfig.auctionIsValid == false,
+            "Auction already marked as valid"
+        );
+        _projectConfig.selloutPrice = _newSelloutPrice;
```

Comment:
Should this method also toggle the value of `auctionIsValid`?

Reply:
AH nevermind–I see why now this is not relevant, given the below method.

Reply:
cool, resolving 🙏